tests/test_utils.py

```diff
@@ -573,21 +573,16 @@ def test_get_sys_path_for_python_executable():
 @pytest.mark.parametrize(
     ("pyproject_param", "new_default"),
     (
-        # From sync
-        ("ask", True),
```

Comment:
FYI: I have removed the options for `pip-sync` because we only test the [pip-compile](https://github.com/jazzband/pip-tools/blob/776c3fa3d8b2da155eed7eb9a5b164f78937987a/tests/test_utils.py#L620-L621) CLI. We could add a separate test for pip-sync with its own options, but it seems redundant to me.